pkg/linter/linter.go

```diff
@@ -67,9 +67,27 @@ type Linter struct {
 	profiling            bool
 	instrumentation      bool
 	hasCustomRules       bool
+	isPrepared           bool
+
+	preparedQuery *rego.PreparedEvalQuery
 }
 
-var lintQuery = ast.MustParseBody("lint := data.regal.main.lint")
+var (
+	lintQueryStr         = "lint := data.regal.main.lint"
+	enabledRulesQueryStr = `[rule |
```

Comment:
💭 I like the momentum here, to do more stuff in Rego. But if it's inline query strings like that, it'll stay under the radar of our beloved linter itself. I suppose we could create some extra files and `go:embed` them...

Reply:
Hmm, that's a great idea actually.. and I'd much prefer these to be in .rego files somewhere. I don't think we can parse Rego files without a package though? Or maybe we can, but Regal doesn't do that currently.

OTOH, I expect both of the "enabled rules" queries to be dropped shortly, as we consolidate on a single prepared query and use input attributes to determine the route.